projects/backend/utils/crossword.ts

```diff
@@ -56,17 +56,22 @@ type CrosswordArticleOverrides = Pick<
 > &
     Pick<CrosswordArticle, 'crossword'>
 
-const shouldHaveSolution = (type: CrosswordType) =>
-    ![CrosswordType.EVERYMAN, CrosswordType.PRIZE].includes(type)
-
 const removeSolutionsFromEntries = (entries: Crossword['entries']) =>
     entries.map(omit(['solution']))
 
 export const patchCrossword = (
     crossword: Crossword,
     type: CrosswordType,
 ): Crossword => {
-    const solutionAvailable = shouldHaveSolution(crossword.type)
+    // for crosswords with a dateSolutionAvailable in the future, hide solutions
+    // this will be permanent even after the date has passed unless the edition is republished
+    // editions might be pressed before a crosssord is due to be published so + 1 day
```

Comment:
yes but isn't 24h an arbitrary measure? Perhaps we should just never send solutions if there is a `dateSolutionAvailable` provided.